### ISSUE TITLE
Why my distillation results are so bad?

### ISSUE THREAD

**Issue by fake-warrior8** [microsoft/vert-papers#15]
I ran the source code 5 times and got the results for  mBERT-TLADV: F1 of de/es/nl  72/76/81, after KD the result becomes 73/76/81, which is much low than the reported result in the original paper (mBERT-TLADV 74/77/81, after KD 75/79/83). Are there any other trick to reproduce the reported results other than the given code?

**Comment by Weile-Chen** [comment]
HI, we don't use other tricks. You can run the source code in the same environment as us.

Pytorch 1.9.0 
Cuda 10.2 
Transformer 3.2.0
Ubuntu 18.0

**Comment by fake-warrior8** [comment]
> HI, we don't use other tricks. You can run the source code in the same environment as us.
> 
> Pytorch 1.9.0 Cuda 10.2 Transformer 3.2.0 Ubuntu 18.0

My environment is the same as yours, except for the Ubuntu version. I don't think the Ubuntu version will have a big impact on the model.

**Comment by Weile-Chen** [comment]
The distillation results are bad because the mBERT-TLADV doesn't have significant improvement.
In our experiment, we can get a mBERT-TLADV with good performance.

Could you share more details? 
Such as the experiment setting, random seeds, and the training log. It will be helpful.

**Comment by fake-warrior8** [comment]
> 

The experiment setting is CentOS7.4, Pytorch1.9, Cuda10.2, Transformer3.2.0, python3.6.9(Only the Linux version is not the same as your setting), the random seeds are the ones shown in your run.sh. To run the run.sh, I only changed the data_path and model path to absolute paths. 
The training log of a certain run is
The num examples of the dev/test set of de is 2867/3005  
The dev F1 list of de = [73.97, 77.54, 75.99, 77.87, 77.22, 76.84, **78.73**, 77.73, 78.65, 78.06]
The test F1 list  of de= [67.73,70.75, 70.33, 70.60, 70.85, 70.12, **71.37**, 70.74, 72.13, 71.69]

**Comment by Weile-Chen** [comment]
Please check your dataset. 

Following other research works, in our experiments, the num examples of the German dev/test set are 3,068 and 3,160.
You can check the dataset details in our paper.

**Comment by fake-warrior8** [comment]
> Please check your dataset.
> 
> Following other research works, in our experiments, the num examples of the German dev/test set are 3,068 and 3,160. You can check the dataset details in our paper.

Where can I download the CoNLL2003 de dataset. The LDC link in the [CoNLL2003 website](https://www.clips.uantwerpen.be/conll2003/ner/) is a non-existent page and I can not find the CoNLL2003 dataset in [LDC](https://catalog.ldc.upenn.edu/search).

**Comment by flydzy** [comment]
Hi, I have the same problem about the dataset, how to get the correct dataset of en and de and divide it into dev and test correctly,could you please tell me more about the details

**Comment by cdxzyc** [comment]
Hello, are there any parameters to be modified when reproducing the paper Advpicker? I use the same conll2002 and conll2003 datasets on wihich can obtain a similar performance to the paper Single-Multi, but when reproducing the Advpicker, I only get F1 scores of 40-50 on 5 random seeds and 3 languages. I ran the run.sh file without any modification.

**Comment by iofu728** [comment]
> Hi, I have the same problem about the dataset, how to get the correct dataset of en and de and divide it into dev and test correctly,could you please tell me more about the details

Hi @flydzy, you can follow the [official preprocess file](https://www.clips.uantwerpen.be/conll2003/ner/000README).

**Comment by iofu728** [comment]
> Hello, are there any parameters to be modified when reproducing the paper Advpicker? I use the same conll2002 and conll2003 datasets on wihich can obtain a similar performance to the paper Single-Multi, but when reproducing the Advpicker, I only get F1 scores of 40-50 on 5 random seeds and 3 languages. I ran the run.sh file without any modification.

Hi @cdxzyc, does any error in your training processing? It seems that there's some wrong with your training. Could you show us more detail so that we can help you to fix it?

**Comment by tellarin** [comment]
Closing for inactivity. We can re-open if needed once there's more details.
